src/devices/Ad7193/Metadata/ISpiDeviceMetadata.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Device.Spi;
+using System.Text;
+
+namespace Ad7193.Metadata
```

Comment:
please remove all interfaces with just one implementation. If you'd like the interface to get added for all ADCs please create separate issue.